<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicHistogram</class>
 <widget class="QWidget" name="PicHistogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="rgbLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#af0000;&quot;&gt;R&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#00b403;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#000cb5;&quot;&gt;B&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; Histogram&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoWithEditboxes" name="rgbHist" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="redLable">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#af0000;&quot;&gt;Red&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; Histogram&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoWithEditboxes" name="redHist" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="greenLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#00b403;&quot;&gt;Green&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; Histogram&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoWithEditboxes" name="greenHist" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blueLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#000cb5;&quot;&gt;Blue&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; Histogram&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoWithEditboxes" name="blueHist" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoWithEditboxes</class>
   <extends>QWidget</extends>
   <header>histowitheditboxes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
